src/components/PageSettingsModal/PageSettingsSchema.jsx

```diff
@@ -32,9 +33,27 @@ export const PageSettingsSchema = (existingTitlesArray = []) =>
         "Title is already in use. Please choose a different title.",
         (value) => !_.includes(existingTitlesArray, value)
       ),
-    permalink: Yup.string().when("layout", (layout, schema) =>
-      layout !== "file"
-        ? schema
+    permalink: Yup.string().when("layout", (layout, schema) => {
+      switch (layout) {
+        case "file":
+          return schema
+        case "post":
+          return schema
+            .required("Permalink is required")
+            .min(
+              PAGE_SETTINGS_PERMALINK_MIN_LENGTH,
+              `Permalink must be longer than ${PAGE_SETTINGS_PERMALINK_MIN_LENGTH} characters`
+            )
+            .max(
+              PAGE_SETTINGS_PERMALINK_MAX_LENGTH,
+              `Permalink must be shorter than ${PAGE_SETTINGS_PERMALINK_MAX_LENGTH} characters`
+            )
```

Comment:
I think this might be quite a constraint for external URLs. For instance, one of the MND's press releases would not be able to be linked because it is 224 characters:

https://www.mnd.gov.sg/newsroom/press-releases/view/15th-meeting-of-the-malaysia-singapore-joint-ministerial-committee-for-iskandar-malaysia-focuses-on-enhancing-investment-and-innovation-ecosystem-to-drive-economic-recovery

Should we lift this limit for external links?